webview/src/stories/Plots.stories.tsx

```diff
@@ -193,25 +193,20 @@ VirtualizedPlots.args = {
 VirtualizedPlots.parameters = chromaticParameters
 
 export const ZoomedInPlot = Template.bind({})
-ZoomedInPlot.parameters = {
-  chromatic: { delay: 300 }
-}
 ZoomedInPlot.play = async ({ canvasElement }) => {
-  const canvas = within(canvasElement)
-  const plots = await canvas.findAllByTestId(/^plot_/)
-  const plot = await waitFor(() => within(plots[0]).getByRole('button'))
+  const plots = await within(canvasElement).findAllByTestId(/^plot_/)
+  const plot = await within(plots[0]).findByRole('button')
 
-  plot.click()
+  userEvent.click(plot)
 }
 
 export const MultiviewZoomedInPlot = Template.bind({})
-MultiviewZoomedInPlot.parameters = {
-  chromatic: { delay: 300 }
-}
 MultiviewZoomedInPlot.play = async ({ canvasElement }) => {
-  const canvas = within(canvasElement)
-  const plot = await canvas.findByTestId('plots-section_template-multi_1')
-  const plotButton = await waitFor(() => within(plot).getByRole('button'))
+  const plot = await within(canvasElement).findByTestId(
+    'plots-section_template-multi_1'
+  )
+  await within(plot).findByRole('graphics-document')
```

Comment:
This waits until the initial plot is rendered before we click on it, and fixes the weird Storybook issue we were having where the multiview plot disappeared. I think that bug was more due to vega instead of storybook, and the new Storybook config is just bringing a symptom to surface.